src/api_methods.py

```diff
@@ -162,19 +179,23 @@ def download_cashflows(user_data, date_start, date_end, path_account):
     """
     # Parse dates
     date_start = pd.to_datetime(date_start)
-    date_end   = pd.to_datetime(date_end)
+    date_end = pd.to_datetime(date_end)
 
     # Format download URL
-    url_account_formated = url_account.format(int_account = user_data['intAccount'], 
-                                            session_id  = user_data['sessionId'], 
-                                            day_i       = date_start.strftime('%d'),
-                                            month_i     = date_start.strftime('%m'),
-                                            year_i      = date_start.strftime('%Y'),
-                                            day_f       = date_end.strftime('%d'),
-                                            month_f     = date_end.strftime('%m'),
-                                            year_f      = date_end.strftime('%Y'))
+    url_account_formated = url_account.format(
+        int_account=user_data["intAccount"],
+        session_id=user_data["sessionId"],
+        day_i=date_start.strftime("%d"),
+        month_i=date_start.strftime("%m"),
+        year_i=date_start.strftime("%Y"),
+        day_f=date_end.strftime("%d"),
+        month_f=date_end.strftime("%m"),
+        year_f=date_end.strftime("%Y"),
+    )
```

Comment:
Same here, I struggle to read the variables when they are compacted like this.

Reply:
Of course, the vertical alignment is your convention if you want to follow in this way I can deal with this. However, I would like note some ideas:

## About vertical alignment

1. what is better is **subjective**.
1. is not an standard
1. If you refactor the code (p.e: rename a variable) you must check that all vertical alignments are corrects.
1. It's ok in a develop phase but I **prefer** avoid it in production or stable code (master).
1. There are tools like black to auto-format code following pep8.

## About the docstring

1. what is better is subjective.
1. Docstrings are written for users (mainly).
1. `help(function)` prints the docstring for a function or `function?` under ipython.
1. We are following numpydocs?

The benefits following a standard are that everyone can google them and write code in the same way. If you want use another conventions, it's ok too! if this is the case I recommend you [write them in contribution guidelines](https://help.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors).

Reply:
Okey, you have convinced me with the develop branch and the automatic tool to convert code to pep8 👍 From now on I will use the develop branch to refactor there my code. 

> what is better is subjective.

Yes. I wanted to share the motivation to stray away from the usual convention. 

> help(function) prints the docstring for a function or function? under ipython.

More than for the notebooks, it is while developing that I read those docstrings.

> We are following numpydocs?

I guess we are from now on ;)